Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -90,26 +92,13 @@ Color StyleProperties::color_or_fallback(CSS::PropertyID id, const DOM::Document
     return value.value()->to_color(document);
 }
 
-void StyleProperties::load_font() const
+void StyleProperties::load_font(Layout::Node const& node) const
 {
-    auto family_value = string_or_fallback(CSS::PropertyID::FontFamily, "Katica");
     auto font_size = property(CSS::PropertyID::FontSize).value_or(IdentifierStyleValue::create(CSS::ValueID::Medium));
     auto font_weight = property(CSS::PropertyID::FontWeight).value_or(IdentifierStyleValue::create(CSS::ValueID::Normal));
 
-    auto family_parts = family_value.split(',');
-    auto family = family_parts[0];
-
-    auto monospace = false;
-    auto bold = false;
-
-    if (family.is_one_of("monospace", "ui-monospace")) {
-        monospace = true;
-        family = "Csilla";
-    } else if (family.is_one_of("serif", "sans-serif", "cursive", "fantasy", "ui-serif", "ui-sans-serif", "ui-rounded")) {
-        family = "Katica";
-    }
-
     int weight = 400;
```

Comment:
I think it would be somewhat nicer to have these hard-coded values (the 700 and 900 below as well) somewhere central. 
Can't we use some `CSS::InitialValues` or similar? 
I mean if the spec hard-codes these values we might as well, but it seems cleaner to move the values out of here.

Reply:
Thanks! I wasn't even aware of `InitialValues`. That makes a lot of sense.

Reply:
Also, just noticed we have a [font-weight enum](https://github.com/SerenityOS/serenity/blob/833936f3ecc635d39c5fa092e88e2056ed749089/Userland/Libraries/LibGfx/FontDatabase.h#L19) which would be good to use too, for clarity.

Reply:
I couldn't get my head around using InitialValues here with properties which are numbers/lengths, but default to a keyword. Going to try and keep this PR simple for now.